cluster/common.sh

```diff
@@ -277,14 +285,15 @@ function load-or-gen-kube-basicauth() {
 #   KUBE_VERSION
 function set_binary_version() {
   if [[ "${1}" =~ "/" ]]; then
-    IFS='/' read -a path <<< "${1}"
+    IFS='/' read -a -r path <<< "${1}"
```

Comment:
Hi, `read -a -r path` cannot run. Because array is necessary just after `-a`. I think `read -ra` is appropriate.

Reply:
thanks for yours comment, i have modified it.